docs/content/Configuration/Config.md

```diff
@@ -478,8 +478,8 @@ using `CUBEJS_JWK_URL`.
 
 #### key
 
-A JSON string that represents a cryptographic key and its' properties. Can also
-be set using `CUBEJS_JWK_KEY`.
+A JSON string that represents a cryptographic key. Similar to `API_SECRET`, which will be deprecated in the feature. Can also
```

Comment:
```suggestion
A JSON string that represents a cryptographic key. Similar to `API_SECRET`, which will be deprecated in the future. Can also
```

Reply:
I assume this is what you meant? If you meant `feature`, I think you can make this simpler by just saying `which will be deprecated`.

Reply:
+1 to @rpaik's suggestion. Also, we should add `API_SECRET` to  `DEPRECATION.md`, but let's do that as part of a separate PR